pingdom-checks/appeals.ping

```diff
@@ -21,15 +21,5 @@ pingdom save-check \
   -a url="/services/appeals/v0/healthcheck" \
   -a apikey="$SANDBOX_APPEALS_API_KEY" \
   -a group="appeals" \
-  -a userids_csv="$NO_USERS" \
-  -a integrationids_csv="$HEALTH_APIS_SLACK_ID"
-
-pingdom save-check \
-  --template request-with-apikey \
-  -a name=dev-appeals-v0-health-check \
```

Comment:
I dont think we want these removed, just not integrated with statuspage.

Reply:
Correct, not removing them just  yet, that is part of SHANK-550 which is on hold until given the all clear to remove them by devops.

Reply:
for clarity ... we're not going to remove the checks against `dev-api.va.gov`. Those will remain in Pingdom. We will remove the Development environment from Statuspage. Dev-api is now just for us so the general public does not need to know about it.